Updated Readme for TDX Release (#81)
salesforcecli/CLI-Unification-demo: README.md

# Develop Across Salesforce With One CLI

> Welcome to the Salesforce CLI Unification Demo CLI repo. We’re designing a new taxonomy and structure for Salesforce CLI with the intention of unifying all of Salesforce’s products under one CLI. Our goal is to provide a seamless developer experience when building applications on any Salesforce product as well as across products. The demo CLI is an interactive simulation setup with the new taxonomy and common workflows, it doesn't actually do anything or change your environment. Only a handful of commands have full help content. Please test it out and tell us what you think.

Reviewer: ```suggestion
Welcome to the Salesforce CLI Unification Demo CLI repo. We’re designing a new taxonomy and structure for Salesforce CLI with the intention of unifying all of Salesforce’s products under one CLI. Our goal is to provide a seamless developer experience when building applications on any Salesforce product as well as across products. The demo CLI is an interactive simulation that implements the new taxonomy and common workflows, although it doesn't actually do anything or change your environment. Only a handful of commands have full help content. Please test it out and tell us what you think.
```

## Strategy

> See Strategy and Structure for more details 

Reviewer 2: ```suggestion
See (Strategy and Strucuture)[STRATEGY.md] for more details.
```

## Install

```bash
npm install --global @salesforce/sf-demo

> sf help

Reviewer: This doesn't work right now (although maybe it will later when this demo cli fully implements the recent help changes?).  This works today:

sf --help

```
## Feedback

We built this demo CLI because we want feedback from the Salesforce Developer Community on the new structure. We also want to know whether our new design feels natural and is what you'd expect in a unified CLI experience. To provide feedback, use the issues tab in this repository.

Here’s the type of feedback we’re looking for:

* Overall impressions.
* How your experience could be improved. In particular:
    * Was it easy to navigate?

>     * The commands: `project deploy`, `sf login`, and `env list` (along with a few others) have the updated help structure and output, is the help output useful?

Reviewer: ```suggestion
    * The commands: `sf project deploy`, `sf login`, and `sf env list` (along with a few others) have the updated help structure and output. Is it useful?
```

    * `sf login`, `sf env list`, and `sf project deploy` all simulate the cross cloud experience. Is this what you expected? What did you like? How could they be better? 
* Are there `sfdx` commands that you use often but are missing from this demo CLI?  (Note that this demo CLI doesn’t include all the commands that we plan to have in `sf`. )
* What did you like?
* What could be better?

## Demo

There are two types of commands in this demo:

1. Commands that simulate real workflows.
2. Static commands provided to showcase the overall design, structure, and direction.

*Simulation commands*
Try running running the following commands:

* sf login
* sf project deploy
* sf env list
* sf whoami

*Static commands*

> These topics are speculative and are provided as an example of the types of topics that we expect to deliver, please review and provide feedback. Run --help  with different levels of the demo CLI to navigate the tree, i.e. sf --help shows the top-level topics and commands, sf project --help shows the subtopics and commands under "project", etc.

Reviewer: ```suggestion
These topics are speculative and are provided as an example of the types of topics that we expect to deliver. Please review them and provide feedback. Run --help  with different levels of the demo CLI to navigate the tree. For example, `sf --help` shows the top-level topics and commands, `sf project --help` shows the subtopics and commands under `project`, and so on.
```
